### DISCUSSION TITLE
playlist view jerks on websocket reconnect

### DISCUSSION THREAD

**Discussion by L0ric0** [jcorporation/myMPD#485] [Ideas]
myMPD version: 7.0.2

**Describe the bug**
when in the playlist view and the websocket disconnects the view jerks because on reconnect the top song has the number 1, is not bold and only the duration of the song is shown (and not the remaining time), shortly after the song is presented in bold, the 1 changes to a play simbol and the remaining time of the song is added

**To Reproduce**

1. Go to playlist view
2. wait for websocket to disconnect
3. see the jerking

second way probably same reson:
1. go tot playlist view
2. open playback settings
3. wait a bit and see the table jerk
4. save the settings and see the table jerk again

**Expected behavior**

no jerking because of the table resising (maybe cache the play state and current time and use this for the beginning)

**Server plattform (please complete the following information):**
 - OS: Arch
 - OS version 5.11.4-1-ARCH #1 SMP Sun Mar 7 23:46:10 UTC 2021 aarch64 GNU/Linux
 - MPD version: 0.22.6

**Client plattform (please complete the following information):**
 - Device: Desktop
 - OS: Windows 10
 - Browser firefox
 - Browser version 88
 
 **Additional context**

**Comment by jcorporation** [comment]
This issue will be addressed in #474 for v8.2.0 release.
